system-variables.md

```diff
@@ -652,6 +652,17 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Default value: `OFF`
 - This variable is used to set whether the `AUTO_INCREMENT` property of a column is allowed to be removed by executing `ALTER TABLE MODIFY` or `ALTER TABLE CHANGE` statements. It is not allowed by default.
 
+### tidb_analyze_partition_concurrency
+
+> **Warning:**
+>
+> The feature controlled by this variable is not fully functional in the current TiDB version. Do not change the default value.
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: `1`
```

Comment:
Is there a range for this variable?

Reply:
We do not recommend users to set it before the related feature is released. So the value range is not necessary IMO.

Reply:
From the code in `sessionctx/variable/sysvar.go`:
```go
        {Scope: ScopeGlobal | ScopeSession, Name: TiDBAnalyzePartitionConcurrency, Value: strconv.FormatInt(DefTiDBAnalyzePartitionConcurrency, 10),
                MinValue: 1, MaxValue: uint64(config.GetGlobalConfig().Performance.AnalyzePartitionConcurrencyQuota), SetSession: func(s *SessionVars, val string) error {
                        s.AnalyzePartitionConcurrency = int(TidbOptInt64(val, DefTiDBAnalyzePartitionConcurrency))
                        return nil
                }},
```

And in `config/config.go`:
```go
        AnalyzePartitionConcurrencyQuota uint   `toml:"analyze-partition-concurrency-quota" json:"analyze-partition-concurrency-quota"`
...
                AnalyzePartitionConcurrencyQuota: 16,
```

So looks like the max is set to the value of `analyze-partition-concurrency-quota` which seems to be 16 by default.

I would be ok to leave this out of the docs for the reason that @TomShawn just said.

Reply:
@TomShawn I assume the `analyze-partition-concurrency-quota` config is added to the docs in another PR?

Reply:
Yes. I'll ask the related developer to provide the doc input for `analyze-partition-concurrency-quota` in another PR.